business/jaeger.go

```diff
@@ -0,0 +1,261 @@
+package business
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/gorilla/mux"
```

Comment:
Usually we try to have a clear separation between handlers layer and business layer, and keep any of gorilla/mux and such dependency out of the business layer. Could we do the same here?
It means that business functions would not take *Request* argument or such, but more explicit arguments list instead.

Reply:
ofc, this is a first work I need to clean and order thigns

Reply:
It's still there :)
Due to `GetJaegerTraceDetail`